src/modules/avformat/producer_avformat.c

```diff
@@ -490,13 +490,44 @@ static mlt_properties find_default_streams(producer_avformat self)
     // Allow for multiple audio and video streams in the file and select first of each (if available)
     for (i = 0; i < context->nb_streams; i++) {
         // Get the codec context
+        AVFormatContext *vpxContext = NULL;
         AVStream *stream = context->streams[i];
         if (!stream)
             continue;
         AVCodecParameters *codec_params = stream->codecpar;
         const AVCodec *codec = avcodec_find_decoder(codec_params->codec_id);
         if (!codec)
             continue;
+        int switchToVpxCodec = 0;
```

Comment:
Symbols do not use camelCase in MLT! This also applies to `vpxContext`.  Otherwise, this looks OK.